src/ProjectService.Business/Commands/Task/CreateTaskPropertyCommand.cs

```diff
@@ -1,66 +1,84 @@
 using LT.DigitalOffice.Kernel.Responses;
 using LT.DigitalOffice.ProjectService.Business.Commands.Interfaces;
 using LT.DigitalOffice.ProjectService.Data.Interfaces;
 using LT.DigitalOffice.ProjectService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
 using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
 using LT.DigitalOffice.ProjectService.Validation.Interfaces;
 using Microsoft.AspNetCore.Http;
-using System;
-using System.Collections.Generic;
-using System.Linq;
 
 namespace LT.DigitalOffice.ProjectService.Business.Commands
 {
-    public class CreateTaskPropertyCommand : ICreateTaskPropertyCommand
+  public class CreateTaskPropertyCommand : ICreateTaskPropertyCommand
+  {
+    private readonly IDbTaskPropertyMapper _mapper;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IUserRepository _userRepository;
+    private readonly ICreateTaskPropertyValidator _validator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly ITaskPropertyRepository _taskPropertyRepository;
+
+    public CreateTaskPropertyCommand(
+      IDbTaskPropertyMapper mapper,
+      IAccessValidator accessValidator,
+      ITaskPropertyRepository taskPropertyRepository,
+      IUserRepository userRepository,
+      ICreateTaskPropertyValidator validator,
+      IHttpContextAccessor httpContextAccessor)
     {
-        private readonly IDbTaskPropertyMapper _mapper;
-        private readonly IAccessValidator _accessValidator;
-        private readonly IUserRepository _userRepository;
-        private readonly ICreateTaskPropertyValidator _validator;
-        private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly ITaskPropertyRepository _taskPropertyRepository;
+      _mapper = mapper;
+      _validator = validator;
+      _userRepository = userRepository;
+      _accessValidator = accessValidator;
+      _taskPropertyRepository = taskPropertyRepository;
+      _httpContextAccessor = httpContextAccessor;
+    }
 
-        public CreateTaskPropertyCommand(
-            IDbTaskPropertyMapper mapper,
-            IAccessValidator accessValidator,
-            ITaskPropertyRepository taskPropertyRepository,
-            IUserRepository userRepository,
-            ICreateTaskPropertyValidator validator,
-            IHttpContextAccessor httpContextAccessor)
-        {
-            _mapper = mapper;
-            _validator = validator;
-            _userRepository = userRepository;
-            _accessValidator = accessValidator;
-            _taskPropertyRepository = taskPropertyRepository;
-            _httpContextAccessor = httpContextAccessor;
-        }
+    public OperationResultResponse<IEnumerable<Guid>> Execute(CreateTaskPropertyRequest request)
+    {
+      Guid userId = _httpContextAccessor.HttpContext.GetUserId();
+
+      if (!(_accessValidator.IsAdmin() || _userRepository.AreUserProjectExist(userId, request.ProjectId)))
```

Comment:
can use _httpContextAccessor.HttpContext.GetUserId() right in  .AreUserProjectExist

Reply:
done